src/developer/web-api/overview.md

```diff
@@ -86,13 +86,225 @@ using SSL/TLS (HTTPS) to encrypt communication with clients. Consider this
 a hard requirement in order to provide secure interactions with the Web
 API.
 
-### Two-factor authentication { #webapi_2fa } 
+## Two-factor authentication { #webapi_2fa } 
 
 DHIS2 supports two-factor authentication. This can be enabled per user.
 When enabled, users will be asked to enter a 2FA code when logging in. You
 can read more about 2FA [here](https://www.google.com/landing/2step/).
 
-### OAuth2 { #webapi_oauth2 } 
+## Personal Access Token { #webapi_pat_authentication }
+Personal access tokens (PATs) are an alternative to using passwords for
+authentication to DHIS2 when using the API.
+
+PATs can be a more secure alternative to HTTP Basic Authentication,
+and should be your preferred choice when creating a new app/script etc. 
+
+Since HTTP Basic Authentication is considered unsecure because among other things
+that it uses your username and password in clear text, it might be phased out in 
+future versions and made optional, which means it needs to be explicitly enabled 
+in the configuration.
+
+#### Important security concerns!
+
+Your PAT's will automatically inherit all the permissions/authorizations your
+user has at all time. It is therefore extremely important that you limit the access your
+token has depending on the use-case, see more on how to configure your token.
+
+**If you only want the token to have access to a narrow and specific part of the
+server, it is advised to rather create a new special user that you assign only
+the roles/authorities you want it to have access to.**
+
+
+### Creating a token
+To create a new PAT, you have two choices:
+* A. Create a token in the UI on your account's profile page.
+* B. Create a token via the API
+
+### A. Creating a token on the account's page
+Log in with your username and password, go to your profile page
+(Click top right corner, and chose "Edit profile" from the dropdown).
+On your user profile page, choose "Manage personal access tokens" from the
+left side menu.
+You should now be on the "Manage personal access tokens" page and see the
+text: "You don't have any active personal access tokens".
+Click "Generate new token" to make a new token.
+A "Generate new token" popup will be shown and present you with two choices:
```

Comment:
We should add a screenshot here, and/or link to separate documentation just for the user profile app

Reply:
Added some screenshots for the UI flow.